packages/cashier/src/pages/payment-agent/payment-agent-withdraw-form/payment-agent-withdraw-form.tsx

```diff
@@ -16,13 +16,105 @@ import {
 import { getDecimalPlaces, getCurrencyDisplayCode, validNumber } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
-import ErrorDialog from 'Components/error-dialog';
 import PaymentAgentWithdrawConfirm from '../payment-agent-withdraw-confirm';
+import ErrorDialog from 'Components/error-dialog';
 import PaymentAgentReceipt from '../payment-agent-receipt';
 import './payment-agent-withdraw-form.scss';
 
-const validateWithdrawal = (values, { balance, currency, payment_agent = {} }) => {
-    const errors = {};
+type TValidateWithdrawalValueProps = {
+    payment_method: string;
+    payment_agents: string | undefined;
+    payment_agent: string;
+    amount: string;
+    [key: string]: string | undefined;
+};
+
+type TValidateWithdrawalProps = {
+    balance: string;
+    currency: string;
+    payment_agent?: {
+        min_withdrawal?: string;
+        max_withdrawal?: string;
+        value?: string;
+        text?: string;
+    };
+};
+
+type TRadioProps = {
+    children?: string | JSX.Element | JSX.Element[];
+    field: {
+        value?: string;
+        name?: string;
+        onChange?: () => void;
+    };
```

Comment:
@hamid-deriv Same here, So we can reuse it at lines `60` and `76` and probably other places.